typescript/introduction/tsc-classes-and-interfaces/tsc-classes-and-interfaces-intro.md

```diff
@@ -0,0 +1,127 @@
+---
+author: nem035
+type: normal
+category: must-know
+aspects:
+  - introduction
+links:
+  - '[Play with interfaces and classes](https://www.typescriptlang.org/play/index.html#code/JYOwLgpgTgZghgYwgAgGIBsCe1kG8BQyyA9McnOusjFtAM7IC2ArnWMgBZwBuKcyIOIwiESZMF3bAGAexjIwmAA4o2UUAHNRg4QC5kazQG5RpcpWq0oDFm048+lzEwgSZAE1PjJ9hiBnISnBQQq705CCeRGbuMhB0IADk7FCuzFAgEZgSmqI0mAAUAJT63DLA7iYAvvj4COhwdAwAQsBQ7sjAjEroEMLgDBjYUHjaofqGIBomRAgyIGrMCGAyUAU6EBNg6lNFo0REOXQAdBvIALwCoTPINUT5xfsHcwsyvcfoMhoFogfIAAYAIzaHQAJLgjqdQlVOgx8pp-r9kEUbjUanUGk1kAAREIaeadbq9fpgQZWJ4bLY7aaiF6LZardbjAzbTR7Ah-SFnS4bVF5LCPDnPeZ0N4QD5fH5-Ij-dx4gngrnQ2FOBFIlGiNG1OnsCBwDS9C4CCAAd2QrXaBQARABRfW9K0anXIABWcEBgOgJpkCAA1s4eaacfKQFLkFaAFLuz1Qb1+zBW-Aa-B6g3ih4aszA9rIO1plXwqb4N0er0+-3HDMmGIh5BR0ux8vOaSqotAA){documentation}'
+
+---
+
+# Classes and Interfaces
+
+---
+## Content
+
+Interfaces can be used with classes as their **behavioral contract**.
+
+For example, for a `Bird` and a `Dragon` to become flyers, they must implement the `Flyer` contract.
```

Comment:
can we show a counterexample? maybe create a `Fish` class or smth that doesn't have a `fly` method and that will throw an error